From d917ab9a3ce0e92a863b8040e819755804a529f0 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=D0=A0=D1=83=D1=81=D0=BB=D0=B0=D0=BD=20=D0=98=D0=B6=D0=B1?=
 =?UTF-8?q?=D1=83=D0=BB=D0=B0=D1=82=D0=BE=D0=B2?= <lrn1986@gmail.com>
Date: Mon, 13 Apr 2015 16:14:33 +0000
Subject: [PATCH 12/20] [W32][GTK] Use correct .la files as --library arguments
 for GI scanner

This way it plays well with W32 gobject-introspection library resolution
code.
Without that one would get things like:
ERROR: can't resolve libraries to shared libraries: webkitgtk-3.0, javascriptcoregtk-3.0
---
 Source/WebKit/gtk/GNUmakefile.am | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Source/WebKit/gtk/GNUmakefile.am b/Source/WebKit/gtk/GNUmakefile.am
index 2ba8541..5502f25 100644
--- a/Source/WebKit/gtk/GNUmakefile.am
+++ b/Source/WebKit/gtk/GNUmakefile.am
@@ -311,8 +311,8 @@ WebKit-@WEBKITGTK_API_VERSION@.gir: $(G_IR_SCANNER) JavaScriptCore-@WEBKITGTK_AP
 	     --include=Gtk-@GTK_API_VERSION@ \
 	     --include=JavaScriptCore-@WEBKITGTK_API_VERSION@ \
 	     --include=Soup-2.4 \
-	     --library=webkitgtk-@WEBKITGTK_API_VERSION@ \
-	     --library=javascriptcoregtk-@WEBKITGTK_API_VERSION@ \
+	     --library=libwebkitgtk-@WEBKITGTK_API_MAJOR_VERSION@.@WEBKITGTK_API_MINOR_VERSION@.la \
+	     --library=libjavascriptcoregtk-@WEBKITGTK_API_MAJOR_VERSION@.@WEBKITGTK_API_MINOR_VERSION@.la \
 	     --libtool="$(LIBTOOL)" \
 	     --pkg=gobject-2.0 \
 	     --pkg=gtk+-@GTK_API_VERSION@ \
-- 
1.8.5.3

